requirements.txt

```diff
@@ -1,9 +0,0 @@
-boto3==1.26.141
-cosl==0.0.5
-cryptography==40.0.2
-jsonschema==4.17.3
-ops >= 2.0.0
-pydantic==1.10.8
-pyOpenSSL >= "23.0.0"
-tenacity==8.2.2
-urllib3==1.26.16
```

Comment:
Can you please sync with @dragomirp here.
I would like to keep the common approach for both MySQL and PostgreSQL charms/sources, see
https://github.com/canonical/postgresql-operator/blob/main/requirements.txt